locations/container/pom.xml

```diff
@@ -32,6 +32,32 @@
     </parent>
 
     <dependencies>
+        <dependency>
+            <groupId>org.microbean</groupId>
+            <artifactId>microbean-helm</artifactId>
+            <version>2.12.4-SNAPSHOT</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.yaml</groupId>
+                    <artifactId>snakeyaml</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.errorprone</groupId>
+                    <artifactId>error_prone_annotations</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>io.netty</groupId>
+                    <artifactId>netty-tcnative-boringssl-static</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <!-- Added this dependency explicitly, to calm down IntelliJ IDEA and also upgrade this -->
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <version>2.0.30.Final</version>
+            <classifier>${os.detected.classifier}</classifier>
```

Comment:
This looks questionable -- we need to build cross-platform eg when we release.  I think this breaks that requirement.

ISTR there is an "all platforms" version of the JAR above?

Reply:
I'll look into this soon: https://groups.google.com/forum/#!topic/grpc-io/kqCtpB8T8MY

Reply:
Used the uber jar. Let's see how this goes. :)